cypress/support/pages/backoffice/backoffice-page.ts

```diff
@@ -9,17 +9,22 @@ export class BackofficePage extends AbstractPage {
     cy.visitBackoffice(this.PAGE_URL, options);
   };
 
-  protected interceptTable = (params: InterceptGuiTableParams): void => {
+  public interceptTable = (params: InterceptGuiTableParams, callback?: () => void) => {
     const expectedCount = params.expectedCount ?? 1;
```

Comment:
why not make it `>=0` instead of 1 when not provided?

Reply:
Usually client code expects 1 row to be found.